core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -271,19 +267,6 @@ $dot-table-cell-height: 1.25rem;
             color: $color-palette-secondary;
         }
 
-        &.p-datatable-hoverable-rows {
-            .p-datatable-tbody > tr:not(.p-highlight):hover {
-                background: $color-palette-primary-100;
-                color: $text-color-hover;
-                transition: background-color ease-in $basic-speed;
-            }
-
-            .p-datatable-tbody > tr.disabled-row:hover {
-                background: $color-palette-gray-200;
-                color: $color-palette-gray-700;
-            }
-        }
-
```

Comment:
Does this affect other components?

Reply:
Yes, but the idea in general is not to show hover effect at least that the item row has an action.